Sources/Reporter.swift

```diff
@@ -11,7 +11,7 @@ import SwiftStack
 import SwiftChatSE
 import Dispatch
 
-var reportedPosts = [(id: Int, when: Date, difference: Int)]()
+var reportedPosts = [(id: Int, when: Date, difference: Int, messageID: Int, details: String)]()
```

Comment:
Right, didn't test that. All my tests were when I was running fire only in the dev room.